renku/core/management/config.py

```diff
@@ -37,6 +39,15 @@ def _get_global_config_dir():
     return click.get_app_dir(APP_NAME, force_posix=True)
 
 
+class ConfigFilter(Enum):
```

Comment:
I think we have a couple of places where we should roll out more enum based approaches. I really like this approach, only thing I'm wondering if we should do the same with them as we did with errors. Should we make a `enums.py` for the project and put all enums into one place? That way might be easier to reason about all possible options.